Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
     "components/uniset",
     "resources/testdata",
     "utils/fixed_decimal",
+    "utils/memory_bench",
```

Comment:
Discussion point for the group: is this the directory where we want to add this?

Reply:
In the meeting we decided `tools/memory_bench` instead.

Reply:
I tweaked this slightly to: `tools/benchmark/memory` and `tools/benchmark/macros`